### DISCUSSION TITLE
How to Determine Multi-Week Playoff Matchups

### DISCUSSION THREAD

**Discussion by joeyagreco** [cwendt94/espn-api#470] [Q&A] [answered]
I'm trying to determine from an ESPN League what type of scoring format the playoffs are in.

Common types are:

- 1 week per playoff matchup
- 2 weeks per playoff matchup
- 2 weeks per championship matchup

[Sleeper has something in their API for league settings to determine this.](https://github.com/joeyagreco/sleeper/blob/main/sleeper/enum/PlayoffRoundType.py)

Is there a way of determining that in this library? I haven't been able to find anything.

Thanks!

**Comment by cwendt94** [comment]
Thats a great question, in the League class there is a settings field that looks like [this](https://github.com/cwendt94/espn-api/wiki/Settings-Class) and can be access like `league.settings`. Right now it doesn't have weeks per playoff matchup but I just checked the data from espn and we can add that!

**Reply by cwendt94** [reply, marked as answer]
A new field in the settings object called `playoff_matchup_period_length` gives you the weeks per playoff matchup. This is now in ESPN API Package Version 0.32.0
